package.json

```diff
@@ -194,6 +194,7 @@
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "replace-in-file": "7.0.1",
+    "shx": "^0.3.4",
```

Comment:
That looks promising. Can you fix the version (remove `^`) and run `yarn install` again?

Reply:
done, see commit c0d89362625faf4dd0e00e0fb94df6d57fd4aa3e